app/javascript/components/server-components/SubscriptionManager.tsx

```diff
@@ -92,7 +93,7 @@ type Props = {
   us_states: string[];
   ca_provinces: string[];
   used_card: SavedCreditCard | null;
-  recaptcha_key: string;
+  recaptcha_key: string | null;
```

Comment:
added similar behaviour as used everywhere for bypassing recaptcha in development